README.md

```
@@ -24,6 +24,11 @@ Lokit은 사진을 앨범 단위로 관리하고,
             <br>
             <a href="https://github.com/H0ngJu"><I>H0ngJu</I></a>
         </td>
+        <td>
+            <img src="https://avatars.githubusercontent.com/u/89020004?v=4?size=100" width="100">
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)